components/popover/index.js

```diff
@@ -109,10 +108,9 @@ class Popover extends Component {
 		// Related https://stackoverflow.com/questions/35522220/react-ref-with-focus-doesnt-work-without-settimeout-my-example
 		const focusNode = ( domNode ) => setTimeout( () => domNode.focus() );
 
-		// Find first tabbable node within content and shift focus, falling
-		// back to the popover panel itself.
-		const firstTabbable = focus.tabbable.find( this.contentNode.current )[ 0 ];
-		focusNode( firstTabbable ? firstTabbable : this.contentNode.current );
+		// Focus the popover panel so further tips are easily navigated to via the
+		// keyboard.
+		focusNode( this.contentNode.current );
```

Comment:
I also think it's a good compromise between A11y and UX. These buttons focused on popover open are so ugly and confusing (can't distinguish between what's selected and what's focused sometimes). 

But this is a very impactful change, because we use Popover a lot. I think we need to test all the other usage carefully.

Ellipsis menu, Block Settings Menu, Block Switcher, Table Block's toolbar, inserter (also mobile). I just wanted to mention it and I'll try to test it later.

Reply:
After testing this, it might actually be better to add a prop that disables focus and applies it to the tooltip only. In a lot of other places (especially Block Switcher) you click on a button and you really want focus in the new popover. In the block selector, for instance, it seems silly to have to type tab first to start searching.

Now I think it's better we focus by default (existing behaviour, largely seems to work) with an option to disable. I'll add that, should be easy (but I'll do it tomorrow, I'ma go read a book or something now 😆)